blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/Property.java

```diff
@@ -133,24 +133,21 @@ public void writeTo(Object object, TreePruner pruner, DataWriter writer) throws
         TreePruner child = pruner.accept(object, this);
         if (child==null)        return;
 
-        Object d = writer.getExportConfig().getExportInterceptor().getValue(this,object,writer.getExportConfig());
+        Object d = writer.getExportConfig().getExportInterceptor().getValue(this,object, writer.getExportConfig());
```

Comment:
Why are these changes appearing over here? PR for stapler related changes is already merged.